### PR TITLE
feat: ✨ Hello PG from Quarkus app + DB operator

### PR DESCRIPTION
**Pull request by philippart-s**
This PR is to illustrate how to use Quarkus to query a PG managed database.  
The code use the OVHcloud operator to whitelist the MKS IP.

### CODE REVIEW

File: databases-analytics/databases/hello-pg-java/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - DB's username ➡️ `DB_USERNAME`
+    - DB's password ➡️ `DB_PASSWORD`
+
+### Demo
+  - if needed build the image:
+    - `mvn clean package` to create the jar
+    - build the image: `docker build -f src/main/docker/Dockerfile.jvm -t ovhcom/hello-pg-java:1.0.0 .` (⚠️ Change the tag corresponding to the app version you build ⚠️)
+    - after login to your registry, push the image `docker push ovhcom/hello-pg-java:1.0.0`
+  - create the namespace `hello-pg-java`: `kubectl create ns hello-pg-java`
+  - deploy the Quarkus app: `envsubst < src/main/kubernetes/deploy-app.yml | kubectl apply -n hello-pg-java -f -`, ⚠️ Note the use of `envsubst` to use local environment variable when you run `kubectl apply` command. Off course you can (should?) use _configmap_, _secret_, _helm_, _kustomize_,... to manage this kind og data.
```
Comment (scraly) [inline databases-analytics/databases/hello-pg-java/README.md:31]:
```suggestion
  - deploy the Quarkus app: `envsubst < src/main/kubernetes/deploy-app.yml | kubectl apply -n hello-pg-java -f -`, ⚠️ Note the use of `envsubst` to use local environment variable when you run `kubectl apply` command. Of course you can (should?) use _configmap_, _secret_, _helm_, _kustomize_,... to manage this kind of data.
```

File: databases-analytics/databases/hello-pg-java/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+## Deploy a Quarkus application to access to DBaaS
+
+### General information
+ - 🔗 [Getting started with Kubernetes database operator](https://help.ovhcloud.com/csm/en-gb-public-cloud-databases-database-operator?id=kb_article_view&sysparm_article=KB0058469)
+ - 🔗 [Using the OVHcloud Managed Kubernetes LoadBalancer](https://help.ovhcloud.com/csm/en-gb-public-cloud-kubernetes-using-lb?id=kb_article_view&sysparm_article=KB0050007)
+ - 🔗 [OVH token generation page](https://www.ovh.com/auth/api/createToken?GET=/*&POST=/*&PUT=/*&DELETE=/*)
+ - 🔗 [Quarkus](https://quarkus.io/)
+
+### Set up
+  - A JDK (at least 21)
```
Comment (scraly) [inline databases-analytics/databases/hello-pg-java/README.md:10]:
```suggestion
  - A JDK (at least 21)
```

You need also `helm` and `kubectl` CLI installed :)

File: databases-analytics/databases/hello-pg-java/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                 --set ovhCredentials.applicationSecret=$OVH_APPLICATION_SECRET \
+                 --set ovhCredentials.consumerKey=$OVH_CONSUMER_KEY \
+                 --set ovhCredentials.region=$OVH_ENDPOINT \
+                 public-cloud-databases-operator \
+                 oci://registry-1.docker.io/ovhcom/public-cloud-databases-operator \
+                 --version 0.1.3 \
+                 --namespace ovhcloud
+```
+  - apply the CR: `envsubst < src/main/kubernetes/db-cr.yml | kubectl apply -n ovhcloud -f -`  
+  - if needed, delete the pod to force restarting: `kubectl delete pod <podId> -n hello-pg-java`
```
Comment (scraly) [inline databases-analytics/databases/hello-pg-java/README.md:47]:
```suggestion
  - if needed, delete the pod to force restarting: `kubectl delete pod $(kubectl get pods -l app=quarkus -n hello-pg-java -o jsonpath='{.items[0].metadata.name}') -n hello-pg-java`
```

File: databases-analytics/databases/hello-pg-java/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+### Demo
+  - if needed build the image:
+    - `mvn clean package` to create the jar
+    - build the image: `docker build -f src/main/docker/Dockerfile.jvm -t ovhcom/hello-pg-java:1.0.0 .` (⚠️ Change the tag corresponding to the app version you build ⚠️)
+    - after login to your registry, push the image `docker push ovhcom/hello-pg-java:1.0.0`
+  - create the namespace `hello-pg-java`: `kubectl create ns hello-pg-java`
+  - deploy the Quarkus app: `envsubst < src/main/kubernetes/deploy-app.yml | kubectl apply -n hello-pg-java -f -`, ⚠️ Note the use of `envsubst` to use local environment variable when you run `kubectl apply` command. Off course you can (should?) use _configmap_, _secret_, _helm_, _kustomize_,... to manage this kind og data.
+  - the app is deployed but there is an error in the Quarkus app: `Could not obtain connection to query metadata: java.sql.SQLException: Acquisition timeout while waiting for new connection`
+    - the reason: you need to whitelist the MKS node IP to enable the Quarkus app access to the DB
```
Comment (scraly) [inline databases-analytics/databases/hello-pg-java/README.md:33]:
```suggestion
    - the reason: you need to whitelist the OVHcloud Managed Kubernetes cluster's node IP to enable the Quarkus app access to the DB
```

**Review by scraly — CHANGES_REQUESTED** [review]
Top!
just a few remarks :)

File: databases-analytics/databases/hello-pg-java/README.md (hunk 2 above)
Comment (philippart-s) [reply databases-analytics/databases/hello-pg-java/README.md:10]: fixed with https://github.com/ovh/public-cloud-examples/pull/39/commits/86151f4b7586557c607445909c1c630d67093480

**Comment by philippart-s** [comment]
@scraly PR updated with your reamrks

**Review by scraly — APPROVED** [review]

**Pull request merged**